extensions/amp-timeago/0.1/amp-timeago.js

```diff
@@ -0,0 +1,137 @@
+import {inHI} from './third_party/timeagojs/locales/inHI';
+import {inID} from './third_party/timeagojs/locales/inID';
+import {it} from './third_party/timeagojs/locales/it';
+import {ja} from './third_party/timeagojs/locales/ja';
+import {ko} from './third_party/timeagojs/locales/ko';
+import {ml} from './third_party/timeagojs/locales/ml';
+import {nbNO} from './third_party/timeagojs/locales/nbNO';
+import {nl} from './third_party/timeagojs/locales/nl';
+import {nnNO} from './third_party/timeagojs/locales/nnNO';
+import {pl} from './third_party/timeagojs/locales/pl';
+import {ptBR} from './third_party/timeagojs/locales/ptBR';
+import {ro} from './third_party/timeagojs/locales/ro';
+import {ru} from './third_party/timeagojs/locales/ru';
+import {sv} from './third_party/timeagojs/locales/sv';
+import {ta} from './third_party/timeagojs/locales/ta';
+import {th} from './third_party/timeagojs/locales/th';
+import {tr} from './third_party/timeagojs/locales/tr';
+import {uk} from './third_party/timeagojs/locales/uk';
+import {vi} from './third_party/timeagojs/locales/vi';
+import {zhCN} from './third_party/timeagojs/locales/zhCN';
+import {zhTW} from './third_party/timeagojs/locales/zhTW';
+
+timeago.register('ar', ar);
+timeago.register('be', be);
+timeago.register('bg', bg);
+timeago.register('ca', ca);
+timeago.register('da', da);
+timeago.register('de', de);
+timeago.register('el', el);
+timeago.register('en', en);
+timeago.register('enShort', enShort);
+timeago.register('es', es);
+timeago.register('eu', eu);
+timeago.register('fi', fi);
+timeago.register('fr', fr);
+timeago.register('he', he);
+timeago.register('hu', hu);
+timeago.register('inBG', inBG);
+timeago.register('inHI', inHI);
+timeago.register('inID', inID);
+timeago.register('it', it);
+timeago.register('ja', ja);
+timeago.register('ko', ko);
+timeago.register('ml', ml);
+timeago.register('nbNO', nbNO);
+timeago.register('nl', nl);
+timeago.register('nnNO', nnNO);
+timeago.register('pl', pl);
+timeago.register('ptBR', ptBR);
+timeago.register('ro', ro);
+timeago.register('ru', ru);
+timeago.register('sv', sv);
+timeago.register('ta', ta);
+timeago.register('th', th);
+timeago.register('tr', tr);
+timeago.register('uk', uk);
+timeago.register('vi', vi);
+timeago.register('zhCN', zhCN);
+timeago.register('zhTW', zhTW);
+
+export class AmpTimeAgo extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {string} */
+    this.datetime_ = '';
+
+    /** @private {string} */
+    this.locale_ = '';
+
+    /** @private {string} */
+    this.title_ = '';
+  }
+
+  /** @override */
+  buildCallback() {
+    user().assert(this.element.textContent.length > 0,
+        'Content cannot be empty. Found in: %s', this.element);
+
+    this.datetime_ = this.element.getAttribute('datetime');
+    this.locale_ = this.element.getAttribute('locale') ||
+      this.win.document.documentElement.lang;
```

Comment:
In this scenario timeago.js will simple fallback to using en, which is the default.